CRM/Extendedreport/Form/Report/ExtendedReport.php

```diff
@@ -91,7 +91,7 @@ class CRM_Extendedreport_Form_Report_ExtendedReport extends CRM_Report_Form {
   /**
    * @var string
    */
-  protected $_from;
+  public $_from;
```

Comment:
In MM, @seamuslee001 pointed to https://github.com/civicrm/civicrm-core/pull/23912/files as the impetus (ie if the upstream declaration changes, then the downstream declaration becomes incompatible).

Does it really need the declaration at all -- if the upstream class already declares it?

Here's my concern:

* The combination `core@5.51` + `extendedreport@5.14` do/would be valid (*they agree on `protected`*)
* The combination `core@5.52` + `extendedreport@5.15` will/would be valid (*they agree on `public`*)
* The intermediate combinations (eg `core@5.51`+`extendedreport`@`5.15` or `core@5.52`+`extendedreport`@`5.14`) would be invalid (*they disagree about the property*)

But many sysadmins follow processes where you can only switch one codebase at a time, so they'd have to pass through an invalid state?

If `extendedreport` doesn't declare either way... then you could have a valid upgrade path, right? (eg first upgrade to `extendedreport`@`5.15`, which removes the conflict-prone declaration; then upgrade to `core@5.52` at your leisure).

Reply:
I think that makes sense to me so have pushed a fix for that @totten